.github/workflows/swift-toolchain.yml

```diff
@@ -2620,7 +2620,7 @@ jobs:
   debugging_tools:
     name: Debugging Tools
     needs: [context, compilers, devtools, stdlib, sdk]
-    runs-on: ${{ needs.context.outputs.windows_build_runner }}
+    runs-on: ${{ needs.context.outputs.libs_build_runner }}
```

Comment:
This renaming doesn't make sense - the debug tools are not libraries.

Reply:
Renamed to `main_build_runner`. Same as the other comment, this will make more sense once #793 has landed.

Reply:
The renaming here is generally reasonable, as `windows_build_runner` isn't great. I think that `default_build_runner` is a better name though.

Reply:
Renamed.